### ISSUE TITLE
[BUG] When closing result details, focus always gets returned to the action group - view results

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6928]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When you close the result details the focus always gets sent back to the "view result" button in the action tab instead of the original place it was opened from. 

https://github.com/CDCgov/prime-simplereport/assets/10108172/6772036c-3272-4612-814b-ecf4081199a9

### Steps To Reproduce

1. Go to results tab
2. Click a user's name in the results tab
3. Close the results modal
4. See that focus is sent to the incorrect place.

### Expected Behavior

Focus should be returned to the original place that it was opened from.

### Does this issue have a priority label?

- [X] I have added a priority label

### Has issue been assigned to a team?

- [X] I have added a team label

**Comment by johanna-skylight** [comment]
Ticket is currently block by #5603 which is doing a refactor of the view test results page and the action modals

**Comment by johanna-skylight** [comment]
This ticket is good to be worked on now
